content/release-10-17-0/edge-10-17-0-bundle/10_17_0.md

```diff
@@ -0,0 +1,48 @@
+---
+weight: 60
+title: Release 10.17.0
+layout: redirect
+---
+
+The Cumulocity IoT Edge release 10.17 includes new features, improvements, fixes and known issues for the Cumulocity IoT Platform 10.17. For more information, see [What's new in Cumulocity IoT 10.17](https://www.youtube.com/watch?v=43xcs8iUlOs&list=PLexm-BA6Wrr9nL67HHJLHw3vZBlhTy922).
+
+Download the Cumulocity IoT Edge VM based distribution version 10.17 from the [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
+
+## Cumulocity IoT Edge Appliance VM
```

Comment:
We do not use heading level 2, please change all headings accordingly.

Reply:
Done.